src/hotspot/share/memory/metaspace/testHelpers.cpp

```diff
@@ -78,7 +78,7 @@ MetaspaceTestContext::MetaspaceTestContext(const char* name, size_t commit_limit
                     reserve_limit, Metaspace::reserve_alignment_words());
   if (reserve_limit > 0) {
     // have reserve limit -> non-expandable context
-    _rs = ReservedSpace(reserve_limit * BytesPerWord, Metaspace::reserve_alignment(), os::vm_page_size());
+    _rs = ReservedSpace(reserve_limit * BytesPerWord, Metaspace::reserve_alignment(), os::vm_page_size(), mtMetaspace);
```

Comment:
I would make this mtTest. This should not increase the metaspace counters in NMT

Reply:
Done.

Reply:
Huh! Now these comments are appearing saying it should be `mtTest` ??? I think github is messing things up here with hidden comments that make the actual time flow invisible.

@tstuefe  can you please clarify what this should be. Thanks.